source/gui/__init__.py

```diff
@@ -572,8 +571,15 @@ def __init__(self, parent):
 		# Translators: The title of the Welcome dialog when user starts NVDA for the first time.
 		super(WelcomeDialog, self).__init__(parent, wx.ID_ANY, _("Welcome to NVDA"))
 		mainSizer=wx.BoxSizer(wx.VERTICAL)
-		welcomeText = wx.StaticText(self, wx.ID_ANY, self.WELCOME_MESSAGE)
-		mainSizer.Add(welcomeText,border=20,flag=wx.LEFT|wx.RIGHT|wx.TOP)
+		# Translators: The header for the Welcome dialog when user starts NVDA for the first time. This is in larger,
+		# bold lettering 
+		welcomeTextHeader = wx.StaticText(self, label=_("Welcome to NVDA!"))
+		welcomeTextHeader.SetFont(wx.Font(18, wx.DECORATIVE, wx.NORMAL, wx.BOLD))
+		mainSizer.AddSpacer(10)

```

Comment:
I assume this could use `SPACE_BETWEEN_VERTICAL_DIALOG_ITEMS` once the alignment stuff is merged? It'd be good to keep track of this somewhere and fix once alignment is in master.